addon/components/nrg-responsive-takeover.hbs

```diff
@@ -4,8 +4,7 @@
   @isOpen={{true}}
   @dismissable={{false}}
   @takeover={{true}}
-  {{did-insert this.onInsert}}
-  {{will-destroy this.onDestroy}}
+  {{on-resize this.setMainContentStyle}}
```

Comment:
`setMainContentStyle` needs the `@action` decorator to preserve the context of `this`